stylesheets/index.css

```diff
@@ -0,0 +1,267 @@
+body {
+    margin-top: 200px;
+    color: whitesmoke;
+    font-size: 20px;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url("../images/bckg.jpg");
+    position: relative;
+    z-index: 1;
+    max-width: 100%;
+    margin: 0 auto;
+}
+
+.game-links a:link, .game-links a:visited {
+    text-decoration: none;
+    color: white;
+}
+
+.game-links a:hover {
+    text-decoration: underline;
+}
+
+.box {
+    background-color:firebrick;
+    opacity: 0.8;
+    padding: 2%;
+    text-align: justify;
+    font-weight: bold;
+    color: white;
+    margin-left: 12%;
+    margin-right: 13%;
+}
+
+@keyframes tonext {
+    75% {
+        left: 0;
+    }
+    95% {
+        left: 100%;
+    }
+    98% {
+        left: 100%;
+    }
+    99% {
+        left: 0;
+    }
+}
+
+@keyframes tostart {
+    75% {
+        left: 0;
+    }
+    95% {
+        left: -300%;
+    }
+    98% {
+        left: -300%;
+    }
+    99% {
+        left: 0;
+    }
+}
+
+@keyframes snap {
+    96% {
+        scroll-snap-align: center;
+    }
+    97% {
+        scroll-snap-align: none;
+    }
+    99% {
+        scroll-snap-align: none;
+    }
+    100% {
+        scroll-snap-align: center;
+    }
+}
+
+* {
+    box-sizing: border-box;
+    scrollbar-color: transparent transparent; /* thumb and track color */
+    scrollbar-width: 0px;
+}
+
+*::-webkit-scrollbar {
+    width: 0;
+}
+
+*::-webkit-scrollbar-track {
+    background: transparent;
+}
+
+*::-webkit-scrollbar-thumb {
+    background: transparent;
+    border: none;
+}
+
+* {
+    -ms-overflow-style: none;
+}
+
+ol, li {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.carousel {
+    position: relative;
+    margin: 5%;
+    margin-left: 12%;
+    padding-top: 75%;
+    filter: drop-shadow(0 0 10px #0003);
+    perspective: 100px;
+    max-width: 75%;
+    max-height: 75%;
+
+}
+
+.carousel__viewport {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    display: flex;
+    overflow-x: scroll;
+    scroll-behavior: smooth;
+    scroll-snap-type: x mandatory;
+}
+
+.carousel__slide {
+    position: relative;
+    flex: 0 0 100%;
+    width: 100%;
+    /* background-color: #f99; */
+    /* counter-increment: item; */
+    background-image: url("../images/mario2.jpeg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+#carousel__slide1 {
+    background-image: url("../images/mario2.jpeg"); /* Replace with image URL for slide 1 */
+}
+
+#carousel__slide2 {
+    background-image: url("../images/majora2.jpg"); /* Replace with image URL for slide 2 */
```

Comment:
Consider having these images directly in the html in `<img>` tags. This helps with organization by keeping content in the html and styling in the css.